test.c

```diff
@@ -0,0 +1,99 @@
+#include "wslib.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+
+void main(int argc, char *argv[]) {
+
+  /*
+  char tmp_name[200];
+  strcpy(tmp_name,"styl.css");

```

Comment:
`strcpy` to jedna z bardzo niebiezpiecznych funkcji, powinno się używać bezpiecznych alternatyw 

za moich czasów to było najlepsze - https://www.openbsd.org/papers/strlcpy-paper.pdf ale mój C jest rdzawy